CreeDictionary/CreeDictionary/templatetags/url_extras.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+"""
+Get absolute URLs for static assets.
+"""
+from django import template
+from django.urls import reverse
+from django.templatetags.static import StaticNode
+
+register = template.Library()
+
+
+class AbsoluteStaticNode(StaticNode):
+    def url(self, context) -> str:
+        path = super().url(context)
+        if not path.startswith("/"):
+            # Assume it's already an absolute URI:
+            return path
```

Comment:
This makes my head hurt to think about. You have tests, so it’s probably ok? 🤷 

In trying to untangle this, to my great and shocking dismay I just learned from [Wikipedia](https://en.wikipedia.org/wiki/URL#Syntax) that the domain name *is* optional and `<img src="http:horror.gif">` does work :O

Given the enormous potential confusion between absolute *paths* which start with a `/` and absolute *URLs* which do not, can we:

 1. Have a block comment defining the terms “absolute path” and “absolute URL,” possibly quoting RFC1808 “Relative Uniform Resource Locators”:

    > Absolute URLs contain a great deal of information which may already be known from the context of the base document's retrieval, including the scheme, network location, and parts of the URL path.

    I know it says RFC 3986 obsoletes 1808, but I don’t think it does so meaningfully, in that the newer, supposedly-better RFC is so abstract that it *doesn’t even contain the term ‘absolute URL’*

  2. Any mention of ‘absolute’ in the code/comments must include a specifier about which absolute it’s referring to, i.e., ‘absolute URL’ or ‘absolute path’